shell/sct_run_batch.sh

```diff
@@ -58,7 +58,7 @@ command_open() {
 # =============================================================================
 
 # Check number of input params
-if [ "$#" -ne 2 ]; then
+if [ "$#" -ne 2 -a "$#" -ne 3 ]; then
```

Comment:
as we are adding more parameters, including optional ones, i think it would make sense to move to `getopts`. @PaulBautin would you be OK refactoring using this feature? you can get some inspiration from the shell script `install_sct`. Thanks a lot for your contribution!

Reply:
I've added getopts.